.github/workflows/benchmark.yaml

```diff
@@ -25,20 +25,21 @@ jobs:
         uses: actions/setup-python@v3
         with:
           python-version: 3.9
-      - name: Install dependencies
-        shell: bash
-        run: |
-      - uses: r-lib/actions/setup-r@v1
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v1
         with:
-          r-version: '3.5.3' # The R version to download (if necessary) and use.
```

Comment:
This was a drive-by change; the 'install dependencies' step doesn't do anything and it breaks the formatting for the rest of the file (although apparently GitHub still parses it).